.eslintrc.json

```
@@ -45,11 +45,12 @@
       "semi": "off",
       "@typescript-eslint/semi": ["error", "never"],
       "@typescript-eslint/type-annotation-spacing": "error",
-      "@typescript-eslint/unbound-method": "error"
+      "@typescript-eslint/unbound-method": "error",
+      "filenames/match-regex": [2, "^[a-z_-].+$", true]
```

This would be a bit closer to the extended [GitHub config](https://github.com/github/eslint-plugin-github/blob/v4.3.7/lib/configs/recommended.js#L21):

```suggestion
      "filenames/match-regex": ["error", "^[a-z0-9-]+(.d|.config|.test)?$"]
```